examples/time_series/goes_hek_m25.py

```diff
@@ -15,7 +15,7 @@
 
 ###############################################################################
 # Let's first grab GOES XRS data for a particular time of interest
-tr = TimeRange(['2011-06-07 04:00', '2011-06-07 12:00'])
```

Comment:
hmmmm this is an issue now I guess - will `tr = TimeRange(['2011-06-07 04:00', '2011-06-07 12:00'])` this now not return the file for this day? it really should - many people will search only for a flare time so this is important!

Reply:
This issue is due to all clients assuming that if a filename contains the timestamp of a file, 
For example goes contain `2011/6/7` in the url, so it start time is considered as `2011/6/7 00:00` since there's no info about `HHMMSS` and there's one file per day. 
Since scraper compares `start_time_in_attr<=start_time_extracted`, so if we do `2011/4/7 4:00`, so file for `2011/6/7` is not returned. What best thing we should be doing? 

One thing in my mind is compare inferred endtime of a file with the end time provided in the attr. If one of them is true, then we will return the file.

Reply:
I agree with @hayesla in that it is customary to return the file for the day being searched, otherwise it would be against the usual user expectations...

Reply:
hmm is there a way to implement within goes.py? such that if the hh:mm is given it will automatically trim them to 00:00?

Reply:
@hayesla this is fixed now with some tweaks in scraper; by adding :func:`get_timerange_with_extractor` in scraper.